### ISSUE TITLE
[Efficiency] User is taken out of the primary flow without notification.

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#78949]
_This ticket was generated during the pilot process for the new VA.gov experience standards. Please tag `@platform-governance-team-members` on Slack if you need assistance interpreting this ticket._

## Product Information
**Team:** Identity Core Team
**Product:** Terms of Use (ToU)
**Feature:** Terms of Use (ToU)

## Findings details
**VA.gov Experience Standard - issue:** User is taken out of the primary flow without notification.
**VA.gov Experience Standard - category:** Efficiency
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility), @erinrwhite (IA)

### Description

Several links on the page open in a new tab without warning users.

Warning about opening in a new tab is a best practice for all users, but it has a disproportionate impact on screen reader users. Most screen readers don't reliably notify users that a new tab has opened, and so a screen reader user may not realize that this change of context has happened.

Likewise, the "Read the VHA notice of privacy practices" link opens a PDF without warning users about the file format. PDFs have a bad reputation among screen reader users --- anecdotally, some screen reader users just refuse to follow links that go to PDFs (even if they're accessible PDFs) unless it's absolutely necessary to complete a task. Being dropped into a PDF unexpectedly can frustrate users.

## Recommended action
Visible text telling users what's about to happen is an easy solution here: `<a href="...">[link text] (opens in a new tab)</a>` and `<a href="...">[link text] (PDF)</a>`. If that's not workable for design reasons, then providing screen reader only text is a sufficient option, eg. `<a href="...">[link text]<span class="sr-only"> (opens in a new tab)</span></a>`.

## References

- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern:** N/A
- **Content Style Guide:** [Links](https://design.va.gov/content-style-guide/links)
- **Context:** N/A
- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by asg5704** [comment]
[This is the reference document](https://docs.google.com/document/d/1S2MTIZ3tklLheHF--xQQau9PnVkVUlNd/edit?usp=sharing&ouid=105984809890077019983&rtpof=true&sd=true) provided to the Identity-team for the Terms of Use content. (Content was modified by @DanielleThierryUSDSVA and @laurwill in compliance with Office of General Counsel (OGC).

I can absolutely add the "opens in a new tab" once OGC & the above people sign off on it and provides the Identity-team with an updated Terms of Use document. But until that time I'm going to be putting this in the "Won't Fix" category.
